### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by e-kwsm**

### CODE REVIEW

File: tools/build_utils/fypp [hunk 1]
```diff
@@ -37,7 +37,7 @@ Python, one usually interacts with the following two classes:
 * `Fypp`_: The actual Fypp preprocessor. It returns for a given input
   the preprocessed output.
 
-* `FyppOptions`_: Contains customizable settings controling the behaviour of
+* `FyppOptions`_: Contains customizable settings controlling the behaviour of
```
Comment (dev-zero) [inline tools/build_utils/fypp]: please send fixes for Fypp directly to https://github.com/aradi/fypp

**Review by dev-zero — CHANGES_REQUESTED** [review]
Changes to Fypp should be sent directly to https://github.com/aradi/fypp and we will pull them from there.

**Comment by dev-zero** [comment]
Some of the `prettify` formatting checks probably fail due to changes made to our prettifier without re-running on the complete tree, see #750 
The changes requested by the `black` formatter on the other hand need to be incorporated.

@mtaillefumier can you please make sure fixes to the `m4` files end up at upstream (and document where those files were pulled from originally)?

Otherwise LGTM, thanks!

**Comment by mtaillefumier** [comment]
do not reformat the m4 files please.

File: tools/build_utils/fypp (hunk 1 above)
Comment (e-kwsm) [reply tools/build_utils/fypp]: Submitted: aradi/fypp#6

**Comment by e-kwsm** [comment]
> do not reformat the m4 files please.

The changes were reverted.

**Comment by oschuett** [comment]
Seems like you added merge commit. With all the reverts the git history is pretty scrambled anyways. Maybe start over with a single clean commit? You can keep this PR open and simply use `git push --force`.

**Comment by e-kwsm** [comment]
CI failure (`./tools/prettify/prettify.py src/pw/ps_wavelet_base.F` on a local machine) is due to:
```diff
diff --git a/src/pw/ps_wavelet_base.F b/src/pw/ps_wavelet_base.F
index 3e773e1730cb..ac7b2f136479 100644
--- a/src/pw/ps_wavelet_base.F
+++ b/src/pw/ps_wavelet_base.F
@@ -699,7 +699,6 @@ CONTAINS
 !> \author S. Goedecker, L. Genovese
 ! **************************************************************************************************
    SUBROUTINE P_multkernel(n1, n2, n3, lot, nfft, jS, i3, zw, hx, hy, hz)
-      !Arguments
       INTEGER, INTENT(in)                                :: n1, n2, n3, lot, nfft, jS, i3
       REAL(KIND=dp), DIMENSION(2, lot, n2), &
          INTENT(inout)                                   :: zw
@@ -758,7 +757,6 @@ CONTAINS
 !> \author S. Goedecker, L. Genovese
 ! **************************************************************************************************
    SUBROUTINE multkernel(nd1, nd2, n1, n2, lot, nfft, jS, pot, zw)
-      !Arguments
       INTEGER, INTENT(in)                                :: nd1, nd2, n1, n2, lot, nfft, jS
       REAL(KIND=dp), DIMENSION(nd1, nd2), INTENT(in)     :: pot
       REAL(KIND=dp), DIMENSION(2, lot, n2), &
```

**Comment by oschuett** [comment]
Interesting!

Prettify leaves the following unchanged:
```
SUBROUTINE P_multkernel(n1, n2, n3, lot, nfft, jS, i3, zw, hx, hy, hz)
       !Argments
       INTEGER, INTENT(in)                                :: n1, n2, n3, lot, nfft, jS, i3
```

but when changed to:

```
SUBROUTINE P_multkernel(n1, n2, n3, lot, nfft, jS, i3, zw, hx, hy, hz)
       !Arguments
       INTEGER, INTENT(in)                                :: n1, n2, n3, lot, nfft, jS, i3
```

prettify suddenly wants to remove the comment line.

This smells like a bug in fpretty. However, I'm not sure if it's worth investigating. So, mentioning @pseewald just in case he is curious.

As for this PR, you can simply delete that comment line and move on.

**Comment by alazzaro** [comment]
I would assume that there is a conflict with FORD comment here...

**Comment by pseewald** [comment]
Interesting indeed. I think this is done on purpose (not by me): https://github.com/cp2k/cp2k/blob/master/tools/prettify/prettify_cp2k/normalizeFortranFile.py#L123-L126

Shall we remove this?

**Comment by oschuett** [comment]
@pseewald, thanks for the quick response. Yes, I think we can remove that. I've opened #826.

**Comment by oschuett** [comment]
Thanks for fixing these spelling mistakes.

I was wondering if we could maybe turn this into a permanent solution. We could compile a list of commonly misspelled words from this and similar PRs (#74, #302, #609). Then we could let [analyze_src.py](https://github.com/cp2k/cp2k/blob/master/tools/conventions/analyze_src.py) check the code against that list automatically.

**Pull request merged**